src/eterna/ui/AnnotationDialog.ts

```diff
@@ -80,11 +80,15 @@ export default class AnnotationDialog extends Dialog<AnnotationData> {
             });
         }
 
+        // Primary dialog contents
+        const inputContainer = new VScrollBox(0, 0);
+        this.addObject(inputContainer, this.container);
+
```

Comment:
I figure you added this in the event that the dialog exceeds the size of the device screen, so that the dialog contents can be scrolled?